app/src/main/java/org/pesmypetcare/mypetcare/services/pet/PetManagerAdapter.java

```diff
@@ -94,8 +94,8 @@ public boolean registerNewPet(User user, Pet pet) {
             Pair<DateTime, Double> entry = pet.getLastWeightInfo();
             Weight libraryWeight = new Weight(entry.first.toString(), entry.second);
             try {
-                ServiceLocator.getInstance().getPetManagerClient().addFieldCollectionElement(user.getToken(),
-                    user.getUsername(), pet.getName(), PetData.WEIGHTS, libraryWeight.getKey(),
+                ServiceLocator.getInstance().getPetManagerClient().addFieldCollectionElement(user.getToken(), user.getUsername(),
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Line is longer than 120 characters (found 129).](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5643405)